tsconfig.json

```diff
@@ -26,9 +26,7 @@
     "compileOnSave": true,
     "buildOnSave": false,
     "files": [
-        "./app/bootstrap.ts",
-        "./test/bootstrapTest.ts",
-        "./test/checkTest.ts"
+        "./app/bootstrap.ts"
```

Comment:
That comes in different MR, that's the thing which I was talking about having merge conflict